prime-dotnet-webapi/Controllers/EnrolleesController.cs

```diff
@@ -126,12 +126,28 @@ public async Task<ActionResult<Enrollee>> CreateEnrollee(EnrolleeCreatePayload p
                 return BadRequest(ApiResponse.BadRequest(this.ModelState));
             }
 
+            string filename = null;
+            if (payload.IdentificationDocumentGuid != null)
```

Comment:
if the user has a BC services card, we should ignore the identification document GUID entirely